images/macos/provision/core/xcode.ps1

```diff
@@ -52,6 +52,13 @@ $xcodeVersions | ForEach-Object {
     Rebuild-XcodeLaunchServicesDb -Version $_.link
 }
 
+# 14.0 and higher doesn't ship tvOS and watchOS, so installing it manually
+$xcodeVersions | Where-Object { $_.link -ge "14.0" } | ForEach-Object {
+    Write-Host "Installing Simulator Runtimes..."
+    Switch-Xcode -Version $_.link
```

Comment:
I think we don't really need to switch Xcode here.
`Switch-Xcode` changes `xcodebuild` symlink but you can invoke xcodebuild for specific Xcode using full path.
We already use this approach for some other invocations. See https://github.com/actions/runner-images/blob/6701203c1b625cf03ff525c8758afc0e9075639e/images/macos/helpers/Xcode.Helpers.psm1#L36

And since we don't need to switch Xcode, what do you think about moving `xcodebuild -downloadAllPlatforms` command to the first loop that we use for downloading and installing Xcode?
```powershell
Write-Host "Installing Xcode versions..."
$xcodeVersions | ForEach-Object -ThrottleLimit $threadCount -Parallel {
```

Reply:
@maxim-lobanov done